lib/src/components/NxThreatIndicator/NxThreatIndicator.tsx

```diff
@@ -21,9 +21,10 @@ const NxThreatIndicator: FunctionComponent<Props> =
           policyThreatLevel != null ? categoryByPolicyThreatLevel[policyThreatLevel] :
           'unspecified',
 
-        className = `nx-threat-indicator nx-threat-indicator--${category}`;
+        className = `nx-threat-indicator nx-threat-indicator--${category}`,
+        title = `${category} threat level indicator`;
```

Comment:
We want the text to flow decently well with what comes after.  How about `threat level ${category}`?

Reply:
Sounds good f7dbeeeaf4eac36f29d4354de5fa8a99df862565